starter-code/src/app/components/my-movie/my-movie.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="movie-container">
+  <div>
+    <img src="{{movieData.poster}}" alt="">
```

Comment:
Now we know that at the beginning `movie` it will be undefined. Don't forget to handle this error with a `*ngIf`